src/GlobalStyles.js

```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from 'styled-components'
+
+const GlobalStyles = createGlobalStyle`
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+*, *::before, *::after {
```

Comment:
Why do you need "::before" and "::after" if it's the same as "*"? Won't you for example declare the changes for "::before" and "::after""?

Reply:
It is not the same and a very common way to reset _all_ HTML elements, even the Pseudo elements.